src/AlgebraicGeometry/ToricVarieties/cohomCalg/special_attributes.jl

```diff
@@ -184,7 +184,7 @@ function all_cohomologies(l::ToricLineBundle)
         
         # read out the result
         stdout = read(out, String)
-        result = [ZZRingElem(parse(Int, c)) for c in split(chop(chop(split(stdout, "{" )[4])), ",")]
+        result = [parse(ZZRingElem, c) for c in split(chop(chop(split(stdout, "{" )[4])), ",")]
```

Comment:
@HereAround FYI

(I also think `function all_cohomologies(l::ToricLineBundle)` could be changed to use `@attr`, i.e. use `@attr Vector{ZZRingElem} function all_cohomologies(l::ToricLineBundle)` plus the corresponding tweaks to its body. But that would involve changing its indentation (or leave it mis-indented) and since it doesn't give any other immediate benefit, I didn't do it.